stix_shifter_modules/msatp/stix_translation/json_to_stix_translator.py

```diff
@@ -0,0 +1,388 @@
+import re
```

Comment:
Why are you using your own json_to_stix_translator file? It's suppose to be generic for all JSON files. If the file `stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py` is missing anything, perhaps your changes can be added there?